sandy/core/cov.py

```diff
@@ -381,6 +382,8 @@ def corr(self):
             coeff = np.true_divide(1, self.std.values)
             coeff[~ np.isfinite(coeff)] = 0   # -inf inf NaN
         corr = np.multiply(np.multiply(cov, coeff).T, coeff)
+        if not corr.max() <= 1 and corr.min() >= -1:

```

Comment:
@AitorBengoechea , this should be tested already in the data setter...just by calling method corr once data is initialized. Then you would have this warning only at the init level and not every time you run .corr